com.unity.formats.fbx/Editor/ConvertToPrefabEditorWindow.cs

```diff
@@ -246,14 +247,121 @@ protected override bool Export()
             return true;
         }
 
+
+        protected override string SessionStoragePrefix { get { return "FbxExporterConvertOptions_{0}"; } }
+
+        public override void ClearSessionSettings()
+        {
+            base.ClearSessionSettings();
+            m_convertToPrefabSettingsInstance = null;
+
+            SessionState.EraseString(string.Format(SessionStoragePrefix, nameof(m_prefabSavePaths)));
+            m_prefabSavePaths = null;
+            SessionState.EraseInt(string.Format(SessionStoragePrefix, nameof(SelectedPrefabPath)));
+            SelectedPrefabPath = 0;
+        }
+
+        protected override void StoreSettingsInSession()
+        {
+            base.StoreSettingsInSession();
+
+            // store Prefab Save Paths
+            if (m_prefabSavePaths == null)
+            {
+                return;
+            }
+            var joinedPaths = string.Join(",", m_prefabSavePaths);
```

Comment:
comma is a valid character in a path; maybe choose semicolon (which is a common path separator) ?

Reply:
Or, SetInt for the number and SetString for each index, creating a name for each one.
```
SettionState.SetInt(string.Format(SessionStoragePrefix, nameof(m_prefabSavePaths), n);
for(int i = 0; i < n; ++i) { SessionState.SetString($"....{i}", m_prefabSavePaths[i]); }
```